FreeRTOS_Sockets.c

```diff
@@ -3407,10 +3407,7 @@ void vSocketWakeUpUser( FreeRTOS_Socket_t * pxSocket )
          * may be used in future versions. */
         ( void ) xFlags;
 
-        if( pvBuffer != NULL )
```

Comment:
If this is only for the zero copy method, should we not try to make this in line with the underlying implementation of zero copy? Maybe we should enclose this in an #if block?

Reply:
Here the same answer as above: developers are already using it.